app/services/users.js

```diff
@@ -40,8 +40,21 @@ exports.findAllUsers = (limit, offset) =>
   User.findAndCountAll({
     limit,
     offset,
-    attributes: ['id', 'firstName', 'lastName', 'email']
+    attributes: ['id', 'firstName', 'lastName', 'email', 'role']
   }).catch(() => {
     logger.info('Error trying to find the users');
     throw errors.databaseError('Error processing request in database.');
   });
+
+exports.changeRole = email =>
+  User.update(
+    { role: 'admin' },
```

Comment:
why this method always change the role to admin? doesnt have to allow to change to any role?